client/js/libs/handlebars/tz.js

```diff
@@ -1,8 +1,10 @@
+var timestamp = window.timestamp;
```

Comment:
If it isn't kept in window, it can't be used here at all (because of minification). When we can properly modularise the code (and pass state between them) then it'll be much better. For just now, I think this is the best solution.

But yeah, of course, that's a good point. It's still early in the morning :-P